src/net.cpp

```diff
@@ -747,7 +747,7 @@ CNetMessage V1TransportDeserializer::GetMessage(const std::chrono::microseconds
     CNetMessage msg(std::move(vRecv));
 
     // store command string, time, and sizes
```

Comment:
I think there are a few comments which could be updated.

```
git grep --ignore-case command ./src/net*

Reply:
Let me take it up as a follow-up.